### ISSUE TITLE
Compose CLI tool 2.26.0 does not offer an update option (latest is 2.27.1)

### ISSUE THREAD

**Issue by odockal** [containers/podman-desktop#7304]
### Bug description

I have installed other (2.26.0) than latest version of the compose cli tools via Podman Desktop. I would expect that I got an update option in the CLI Tools Page to get newest version available. 

![Screenshot_20240527_145815](https://github.com/containers/podman-desktop/assets/19164299/69735964-84ce-4da6-b678-f2b4a51e8467)

### Operating system

Linux, Fedora 40

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

**Comment by jeffmaury** [comment]
Can't reproduce on Win11Pro

**Comment by lstocchi** [comment]
I've installed fedora 40 on hyper-v and it seems working fine for me.
Installed desktop using flatpak, run it, installed an old version of docker-compose through the onboarding, went to the cli page and it showed me the update button. I also tried to update and it worked. Did you do something different @odockal ?
![compose_update](https://github.com/containers/podman-desktop/assets/49404737/f854d79a-53ac-45c4-8b5c-3109dd9c39b6)
